khmer/utils.py

```diff
@@ -18,4 +18,19 @@ def print_error(msg):
     print >>sys.stderr, msg
 
 
+def write_record(record, fp):
+    """
+        Writes output sequence and returns str.
```

Comment:
Oh, I see, you're taking your cue from print_error -- please fix both to have the docstring start at the same indentation as """ ;)